build.sbt

```diff
@@ -6,6 +6,9 @@ import scala.collection.immutable
 
 val testAndCompileDependencies: String = "test->test;compile->compile"
 
+ThisBuild / libraryDependencySchemes +=
+  "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always // due to Identity using lift-json
```

Comment:
This is a nasty hack, but necessary as our Identity libraries use lift-json, and lift-json is compiled against an old version of `scala-xml`.

The hack is used in [many](https://github.com/search?q=org%3Aguardian%20libraryDependencySchemes&type=code) Guardian repos!